src/bci_build/package/gcc.py

```diff
@@ -46,6 +46,7 @@ def _is_main_gcc(os_version: OsVersion, gcc_version: _GCC_VERSIONS) -> bool:
                 (gcc_pkg := f"gcc{gcc_version}"),
                 (gpp := f"{gcc_pkg}-c++"),
                 "make",
+                "gawk",
```

Comment:
Not objecting, just asking, but this is unrelated to the `-s` flag that you added, right?

Reply:
yes. there was a second commit with the rationale added. "awk" is needed by ./configure.